iOS_SDK/OneSignalSDK/OneSignalUser/Source/OSIdentityModel.swift

```diff
@@ -39,6 +39,9 @@ class OSIdentityModel: OSModel {
     }
 
     var aliases: [String: String] = [:]
+    
+    // TODO: We need to make this token secure
+    public var jwtBearerToken: String?
```

Comment:
[Keychain](https://developer.apple.com/documentation/security/keychain_services)?

Reply:
Keychain normally isn't used for JWT. JWTs are used designed to be short lived (normally 1 hour), so I don't think we need to do anything extra to secure it.